packages/venia-ui/lib/components/Routes/routes.js

```diff
@@ -1,30 +1,13 @@
-import React, { Suspense, lazy } from 'react';
+import React, { Suspense } from 'react';
```

Comment:
We're dogfooding this now! Venia's builtin static routes are now attached via the `routes` target.

Reply:
@jcalcaben ^^

Reply:
I agree. I can add that as a sub-task for me when I revise the docs for this PR